### PR TITLE
update colab instructions to deprecate conda, show cudf preinstalled on colab

### PR DESCRIPTION
**Pull request by taureandyernv**
- [x] Add link to RAPIDS cudf.pandas notebook
- [x] Remove conda sections
- [x] update pip sections
~~- [ ] add pip nightly instructions~~ Next PR

### CODE REVIEW

**Review by jacobtomlinson — COMMENTED** [review]
Thanks @taureandyernv, this is super exciting!

File: source/platforms/colab.md [hunk 1]
```diff
@@ -2,15 +2,12 @@
 
 ## Overview
 
-This guide is broken into two sections:
+RAPIDS cuDF is preinstalled on Google Colab and instantly accelerates Pandas with zero code changes.  [You can quickly get started with our tutorial notebook](https://nvda.ws/rapids-cudf). This guide is applicable for users who want to utilize the full suite of the RAPIDS libraries for their workflows.  It is broken into two sections:
```
Comment (jacobtomlinson) [inline source/platforms/colab.md:5]: Remove double space to make linter happy.

```suggestion
RAPIDS cuDF is preinstalled on Google Colab and instantly accelerates Pandas with zero code changes. [You can quickly get started with our tutorial notebook](https://nvda.ws/rapids-cudf). This guide is applicable for users who want to utilize the full suite of the RAPIDS libraries for their workflows.  It is broken into two sections:
```

File: source/platforms/colab.md [hunk 2]
```diff
@@ -2,15 +2,12 @@
 
 ## Overview
 
-This guide is broken into two sections:
+RAPIDS cuDF is preinstalled on Google Colab and instantly accelerates Pandas with zero code changes. [You can quickly get started with our tutorial notebook](https://nvda.ws/rapids-cudf). This guide is applicable for users who want to utilize the full suite of the RAPIDS libraries for their workflows.  It is broken into two sections:
```
Comment (jacobtomlinson) [inline source/platforms/colab.md:5]:
```suggestion
RAPIDS cuDF is preinstalled on Google Colab and instantly accelerates Pandas with zero code changes. [You can quickly get started with our tutorial notebook](https://nvda.ws/rapids-cudf). This guide is applicable for users who want to utilize the full suite of the RAPIDS libraries for their workflows. It is broken into two sections:
```

**Comment by taureandyernv** [comment]
hey @jacobtomlinson , thanks for the updates.  lgtm.  i'll update with nightly installs later

**Review by jacobtomlinson — APPROVED** [review]
Ok perfect, I'll merge this in now.

**Pull request merged**